src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -67,7 +68,22 @@ protected virtual void HandleNamedTypeSymbolStart(SymbolStartAnalysisContext con
         private sealed class CompilationAnalyzer
         {
             private readonly object _gate;
-            private readonly Dictionary<ISymbol, ValueUsageInfo> _symbolValueUsageStateMap;
+            /// <summary>
+            /// State map for candidate member symbols, with the value indicating how each symbol is used in executable code.
+            /// </summary>
+            private readonly Dictionary<ISymbol, ValueUsageInfo> _symbolValueUsageStateMap = new();
+            /// <summary>
+            /// List of properies, which has a 'get' accessor usage, while the property itself is not used, e.g.:
+            /// <code>
+            /// class C
+            /// {
+            ///     private int P { get; set; }
+            ///     public void M() { P++; }
+            /// }
+            /// </code>
+            /// Here 'get' accessor is used in increment operation, but 'P' itself is not used anywhere else, so it can be safely removed
```

Comment:
```suggestion
            /// Here, 'get' accessor is used in an increment operation, but the result of the increment operation isn't used and 'P' itself is not used anywhere else, so it can be safely removed
```